2-structured-data/books/model-datastore.js

```diff
@@ -95,9 +95,9 @@ module.exports = function(config) {
       .order('title')
       .start(token);
 
-    ds.runQuery(q, function(err, entities, cursor) {
+    ds.runQuery(q, function(err, entities, nextQuery) {
       if (err) { return cb(err); }
-      var hasMore = entities.length === limit ? cursor : false;
+      var hasMore = entities.length === limit ? nextQuery.startVal : false;
```

Comment:
+jonparrot 'nextQuery' isn't defined in the version of model-datastore.js that I downloaded. Below is how the list() function is defined. Is 'nextQuery' a global variable? 

function list(limit, token, cb) {
    var q = ds.createQuery([kind])
      .limit(limit)
      .order('title')
      .start(token);

    ds.runQuery(q, function(err, entities, cursor) {
      if (err) { return cb(err); }
      var hasMore = entities.length === limit ? cursor : false;
      cb(null, entities.map(fromDatastore), hasMore);
    });
  }